userguide/jails.rst

```diff
@@ -174,10 +154,10 @@ To open the full jail creation form, click
 
 :numref:`Table %s <jail_basic_props_tab>` summarizes the available
 options of the :guilabel:`Basic Properties` of a new jail. By default,
-the only required values to create a jail is the :guilabel:`Jail Name`
-and :guilabel:`Release`. However, it is recommended to configure these
-basic properties as a simple method to quickly create an immediately
-usable jail. Many more advanced settings are available in the
+the only required values to create a jail are the :guilabel:`Jail Name`
+and :guilabel:`Release`. It is recommended to configure these basic
+properties as a simple method to quickly create an immediately usable
```

Comment:
"immediately-usable", but better to avoid:
...a simple method to quickly create a jail for immediate use.